pandas/core/reshape/concat.py

```diff
@@ -173,6 +173,7 @@ def concat(
         they are all None in which case a ValueError will be raised.
     axis : {0/'index', 1/'columns'}, default 0
         The axis to concatenate along.
+        If 0, the outcome is identical to :meth:`DataFrame.append`.
```

Comment:
`append` is deprecated, we shouldn't mention it here

Reply:
append mention removed